### PR TITLE
Add documentation for Apache Pinot integration

### PR DESCRIPTION
**Pull request by cbalci**
Adding documentation to capture the built-in support added to Apache Pinot in the last few years.

I was able to run Jekyll to validate the formatting:

Screenshot 2024-04-25 at 15 23 52

### CODE REVIEW

File: _includes/toc.html [hunk 1]
```diff
@@ -248,6 +248,7 @@
         <a data-toggle="collapse" class="menu collapsed" href="#collapse_quantiles_studies">Quantiles Studies</a>
       </p>
       <div class="collapse" id="collapse_quantiles_studies">
+        <li><a href="{{site.docs_dir}}/QuantilesStudies/KllSketchVsTDigest.html">•KLL sketch vs t-digest</a></li>
```
Comment (AlexanderSaydakov) [inline _includes/toc.html]: why this was restored? outdated clone, perhaps?

Comment (cbalci) [reply _includes/toc.html]: It [still exists](https://github.com/apache/datasketches-website/blob/master/src/main/resources/docgen/toc.json#L212) in master branch `toc.json`.
I thought it was inadvertently removed so left in the PR, but please let me know if it was supposed to be removed.

Comment (AlexanderSaydakov) [reply _includes/toc.html]: I seems like we forgot to remove it from json, but it is not visible on the web site, which is intentional.

Comment (cbalci) [reply _includes/toc.html]: Sounds good, removed from 'toc.json' and the html file.

**Pull request merged**